CMakeLists.txt

```diff
@@ -0,0 +1,83 @@
+#
+# Distributed Linear Algebra with Future (DLAF)
+#
+# Copyright (c) 2018-2024, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+cmake_minimum_required(VERSION 3.22)
+
+# FIXME: CXX needed by DLA-Future
+# FIXME: C needed by HDF5 (dla-future +hdf5)
+project(DLAF-Fortran VERSION 0.1.0 LANGUAGES Fortran C CXX) # TODO: Remove C and CXX
```

Comment:
🤔 how is it done in other projects?

Reply:
Should DLA-Future be using `enable_language(C CXX)` in its `DLAFConfig.cmake` file perhaps? Or `find_package(HDF5 COMPONENTS C)` or something like that? I'm not sure what the right thing to do is though...

Reply:
I suspect that `FindMPI` does require the `C` and `CXX` languages to be enabled. (`C` only when HDF5 is a dependency of DLA-Future, so we might be able to remove it by making HDF5 private).

Reply:
We will revisit this in the future, together with https://github.com/eth-cscs/DLA-Future/issues/1122.